lib/pbench/server/api/__init__.py

```diff
@@ -148,8 +153,9 @@ def create_app(server_config):
     app.config["TESTING"] = False
 
     api = Api(app)
+    mail = Mail(app)
```

Comment:
Since this isn't used outside your new class, I suggest removing it from this module entirely; import the package and construct the instance inside your `ForgotMail.__init__()` instead.